src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -159,12 +162,28 @@
       }
     }
 
+    &.pf-m-action {
+      --pf-c-dropdown__toggle--m-split-button--child--PaddingRight: var(--pf-c-dropdown__toggle--m-split-button--m-action--child--PaddingRight);
+      --pf-c-dropdown__toggle--m-split-button--child--PaddingLeft: var(--pf-c-dropdown__toggle--m-split-button--m-action--child--PaddingLeft);
+
+      .pf-c-dropdown__toggle-button {
+        /* stylelint-disable max-nesting-depth */
+        &:not(:first-child) {
+          margin-left: var(--pf-c-dropdown__toggle--m-split-button--m-action__toggle-button--not-first-child--MarginLeft);
+
```

Comment:
You could apply `margin-right` to `.pf-c-dropdown__toggle-button` then set 
```
&:last-child { 
  --pf-c-dropdown__toggle--m-split-button--m-action__toggle-button--MarginRight: 0;
}
``` 
which shift specificity to the variable rather than the property, so it would be easier to override.

Reply:
nice!